<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HYRD5CG7086PQW" name="API Test Suite" tests="9" failures="3" timestamp="29 Dec 2019 13:14:48 GMT" time="15.726" errors="0">
  <testcase name="getCityWeatherInformation" time="3.241" classname="testscripts.TC001_GET_WeatherInfo"/>
  <testcase name="registerCustomer" time="1.423" classname="testscripts.TC002_POST_CustomerInfo">
    <failure type="java.lang.AssertionError" message="expected [OPERATION_SUCCESS] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [OPERATION_SUCCESS] but found [null]
	at testscripts.TC002_POST_CustomerInfo.registerCustomer(TC002_POST_CustomerInfo.java:35)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- registerCustomer -->
  <testcase name="googleMapInformatoin" time="0.689" classname="testscripts.TC003_GET_GoogleMapsInfo"/>
  <testcase name="validateCityWeatherInformation" time="1.226" classname="testscripts.TC004_GET_ValidatingJSONResponse"/>
  <testcase name="extractValuesOfEachNodeInJSON" time="1.182" classname="testscripts.TC005_GET_ExtractValuesOfEachNodeInJSON"/>
  <testcase name="testAuthorization" time="1.138" classname="testscripts.TC006_GET_RequestAuthorization"/>
  <testcase name="getSingleUser" time="2.525" classname="testscripts.Testscript_01_EmployeeFeature">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at testscripts.Testscript_01_EmployeeFeature.getSingleUser(Testscript_01_EmployeeFeature.java:81)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- getSingleUser -->
  <testcase name="testPost" time="0.986" classname="testscripts.Testscript_01_EmployeeFeature">
    <failure type="java.lang.IllegalArgumentException" message="accessToken cannot be null">
      <![CDATA[java.lang.IllegalArgumentException: accessToken cannot be null
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:238)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)
	at io.restassured.internal.assertion.AssertParameter.notNull(AssertParameter.groovy:26)
	at io.restassured.internal.assertion.AssertParameter$notNull.callStatic(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:55)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:197)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:217)
	at io.restassured.internal.AuthenticationSpecificationImpl.oauth2(AuthenticationSpecificationImpl.groovy:158)
	at io.restassured.specification.AuthenticationSpecification$oauth2.callCurrent(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:157)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:177)
	at io.restassured.internal.AuthenticationSpecificationImpl.oauth2(AuthenticationSpecificationImpl.groovy:151)
	at testscripts.Testscript_01_EmployeeFeature.testPost(Testscript_01_EmployeeFeature.java:93)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testPost -->
  <testcase name="getAllUsers" time="1.72" classname="testscripts.Testscript_01_EmployeeFeature"/>
</testsuite> <!-- API Test Suite -->
